src/components/NotificationPanel/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./styles.css";
+import NotificationBox from "./NotificationBox";
+
+/* window.onload = function() {
+  console.log("darkness");
+  document.getElementById("overlay").style = "display: block";
+};
+
+window.onunload = function() {
+  console.log("hello");
+  document.getElementById("overlay").style = "display: none";
+}; */
+
+const NotificationPanel = ({ notifications }) => (
+  <div>
+    <div className="overlay" />
```

Comment:
you shouldn't create divs like this just for visual changes. html should have semantic meaning, anything visual comes from css. I think you can create a container around notification panel and apply the background-color to that. It's a subtle change, but I really don't like divs that are empty like this